sample_programs/shufflenetv2_10/image.c

```diff
@@ -0,0 +1,235 @@
+/*
+ * image.c - Sample program for Shufflenet
+ *
+ *
+ *
+ */
+
+#include <stdio.h>
+#include <OnnxMlirRuntime.h>
+#include "json-c/json.h"
+#include <string.h>
+#include <assert.h>
+#include <stdlib.h>
+#include "onnx/onnx_pb.h"
+#include <fstream>
+#include <vector>
+
+#define NUM_INPUTS 1 // When using the same image.c file for a different model, specify the number of inputs here depending on the model
+
+using namespace std;
+
+extern "C" {
+  OMTensorList *run_main_graph(OMTensorList *);
+}
+
+void export_layer_output_to_json(OMTensorList *, char*, char*);
+
+int main(int argc, char *argv[]) {
+
+// Input pointers needed for the model	
```

Comment:
Please add indentation before this comment.

Reply:
Made changes in the commit: [097935c](https://github.com/DependableSystemsLab/LLTFI/pull/18/commits/097935c54300329580a27712ec07b0d2cdb294a3)